src/modules/campaign-monitor/services/campaign-monitor.service.ts

```diff
@@ -203,15 +204,15 @@ export class CampaignMonitorService {
 
 	public async sendTransactionalMail(
 		emailInfo: CampaignMonitorSendMailDto
-	): Promise<void | BadRequestException> {
+	): Promise<void | BadRequestException | boolean> {
```

Comment:
We shouldn't add exceptions in the return type
```suggestion
	): Promise<boolean> {
```